build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
     testImplementation 'com.natpryce:hamkrest:1.8.0.1'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
-}
+    testImplementation 'com.google.code.gson:gson:2.8.8'}
```

Comment:
I think we should use https://github.com/Kotlin/kotlinx.serialization instead of GSON for JSON (de)serialization. It's the standard library for Kotlin.

Let me know what you think